integration/csi/go.sum

```diff
@@ -11,8 +11,6 @@ cloud.google.com/go/firestore v1.1.0/go.mod h1:ulACoGHTpvq5r8rxGJ4ddJZBZqakUQqCl
 cloud.google.com/go/pubsub v1.0.1/go.mod h1:R0Gpsv3s54REJCy4fxDixWD93lHJMoZTyQ2kNxGRt3I=
 cloud.google.com/go/storage v1.0.0/go.mod h1:IhtSnM/ZTZV8YYJWCY8RULGVqBDmpoyjwiyrjsg+URw=
 dmitri.shuralyov.com/gpu/mtl v0.0.0-20190408044501-666a987793e9/go.mod h1:H6x//7gZCb22OMCxBHrMx7a5I7Hp++hsVxbQ4BYO7hU=
-github.com/Alluxio/alluxio-csi v0.0.0-20210426175157-bbf121c94da5 h1:6MsrTti67sjwcni0uzS0xeY2Hjw4Ocv7wsxnV+aVYvA=
-github.com/Alluxio/alluxio-csi v0.0.0-20210426175157-bbf121c94da5/go.mod h1:CTFpOzx9k2rBgkdmxlNI+GXLlBz5AlJgm9slSKSnMCI=
```

Comment:
What is this go.sum file? Do you know @LuQQiu ?

Reply:
It's generated by `go mod` (go package management tool). It contains the version number of go project dependencies. Refer: https://blog.golang.org/using-go-modules#TOC_3.

Reply:
I'm not sure if this should be included into the repo, as it is auto-generated and can change every time. No one will be reviewing it as well. I'm fine with the change in this PR but in the future I recommend we find a way to get rid of it if possible.

Reply:
According to this section: https://blog.golang.org/using-go-modules#TOC_3.
```
The go command uses the go.sum file to ensure that future downloads of these modules retrieve the same bits as the first download, to ensure the modules your project depends on do not change unexpectedly, whether for malicious, accidental, or other reasons. Both go.mod and go.sum should be checked into version control.
```

Golang community recommends to check the file into version control. The file will be used when building go project. Otherwise, the build may use different dependencies in different environments

Reply:
Cool, thanks for the check!